com.ifedorenko.p2browser/src/com/ifedorenko/p2browser/views/InstallableUnitFilterComposite.java

```diff
@@ -105,6 +106,8 @@ public void modifyText( ModifyEvent e )
                 {
                     listener.filterChanged( e2 );
                 }
+            	System.out.println("End modify listener");
```

Comment:
Why do you need a messy console for these notices? Could they be logged more quietly by default, and only turned on for a verbose mode, like the eclipse `-debug` flag?